listener/upload_test.go

```diff
@@ -225,8 +225,8 @@ func TestUpdateSystemRepos1(t *testing.T) {
 
 	systemID := 5
 	rhAccountID := 1
```

Comment:
Please do not bother to change it, only by chance if one happens, but you could do this :smile: 
```suggestion
	systemID := int64(5)
	rhAccountID := int64(1)
```

Reply:
It's used in `updateSystemRepos()` which still expects `int`.